elementary/gtk-3.0/gtk-widgets.css

```diff
@@ -3568,9 +3568,11 @@ infobar.error revealer > box {
     -gtk-icon-palette: error #fff;
 }
 
-infobar.error label {
+infobar.error label,
+infobar.error .close {
     color: #fff;
     text-shadow: 0 1px 1px @error_color;
+    -gtk-icon-shadow: 0 1px 1px alpha (shade (@error_color, 1.4), 0.4);
```

Comment:
This should match the text shadow that is right above it and just be `@error_color`